### PR TITLE
Provide arpack.h/hpp in install directory

### PR DESCRIPTION
**Pull request by fghoussen**
Provide arpack.h/hpp in install directory.

Everything works (cmake, autotools) fine.
The only problem is that make distcheck fails... And I can't fix this: not a autotool guru...

Franck

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -6,6 +6,11 @@ arpack-ng - 3.6.0
    Example of   C/Fortran binding can be found in the TESTS/iso_c_binding_c.c     file.
    Example of C++/Fortran binding can be found in the TESTS/iso_c_binding_cpp.cpp file.
 
+ [ Franck Houssen ]
+ * Add support for ISO_C_BINDING (Fortran std 2003) to improve portability (Fortran/C/C++).
```
Comment (sylvestre) [inline CHANGES]: ?

Comment (fghoussen) [reply CHANGES]: I know git well !... But so much not github (PR/fork and so on). These are my first steps actually with github and I have to say I am not yet really comfortable with it.... Sorry about that.

After you merged my PR, I had to create an upstream remote, merge my clone with upstream, and push force to github: this made history messy (merged into 90fba75 - this is what you see).

The only thing you may be interested in is 71acc79.

Comment (fghoussen) [reply CHANGES]: I reseted-hard/rebased branches of my clone: pushed force back to github.
Should be clearer now... Sorry, really not used to github yet: you may look at 17717b.

**Comment by sylvestre** [comment]
please rebase it

**Comment by fghoussen** [comment]
No need: this is already included in 19ac601. I believe I can close this PR, and, you could (?) merge the 3 last commits 19ac601+79f7ecc+771cbbd into 1 named "Add support for iso_c_binding" for instance (for history clarity ?... But need to push force... Making a decision about that is up to you !).

First, I did a branch (h_hpp) for this PR (provide arpack.h/hpp).

Then, on top of h_hpp, I created another branch (icb_tst) to PR commits you have just merged this morning (improve iso_c_binding coverage = 19ac601). You merged icb_tst (that includes h_hpp) into master: so, this PR (h_hpp) is included in master.

**Comment by sylvestre** [comment]
This is fine, we can keep the history, it isn't critical and not like we have 100 commits per day ;)
thanks!

**Pull request closed without merging**